linkis-engineconn-plugins/engineconn-plugins/elasticsearch/src/main/scala/com/webank/wedatasphere/linkis/engineplugin/elasticsearch/exception/EsParamsIllegalException.scala

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2019 WeBank
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.webank.wedatasphere.linkis.engineplugin.elasticsearch.exception
+
+import com.webank.wedatasphere.linkis.common.exception.ErrorException
+
+/**
+ *
```

Comment:
The author information needs to be removed. Same to other place

Reply:
> The author information needs to be removed. Same to other place

Thanks, i will remove it.